addons/mail_bot/models/mail_bot.py

```diff
@@ -42,15 +42,22 @@ def _get_answer(self, record, body, values, command=False):
             if odoobot_state == 'onboarding_emoji' and self._body_contains_emoji(body):
                 self.env.user.odoobot_state = "onboarding_command"
                 self.env.user.odoobot_failed = False
-                return Markup(_("Great! 👍<br/>To access special commands, <b>start your sentence with</b> <span class=\"o_odoobot_command\">/</span>. Try getting help."))
+                return Markup(
```

Comment:
`Markup` needs to be put around the tags, not the translation value, otherwise this doesn't resolve the issue.

Translation needs to be in `html_escape`.

See `execute_command_help`.